Amplitude/Amplitude.m

```diff
@@ -235,31 +233,28 @@ - (id)initWithInstanceName:(NSString*) instanceName
         self.minTimeBetweenSessionsMillis = kAMPMinTimeBetweenSessionsMillis;
         _backoffUploadBatchSize = self.eventUploadMaxBatchSize;
 
-        _initializerQueue = [[NSOperationQueue alloc] init];
         _backgroundQueue = [[NSOperationQueue alloc] init];
         // Force method calls to happen in FIFO order by only allowing 1 concurrent operation
         [_backgroundQueue setMaxConcurrentOperationCount:1];
-        // Ensure initialize finishes running asynchronously before other calls are run
-        [_backgroundQueue setSuspended:YES];
         // Name the queue so runOnBackgroundQueue can tell which queue an operation is running
         _backgroundQueue.name = BACKGROUND_QUEUE_NAME;
         
-        [_initializerQueue addOperationWithBlock:^{
+        [_backgroundQueue addOperationWithBlock:^{
```

Comment:
I just searched through the code and all `addOperationWithBlock` and `runOnBackgroundQueue` calls are protected with an apiKey check, so they need to call `initializeApiKey` before any of those methods work.